README.md

```diff
@@ -488,6 +488,37 @@ state_map:
     label: Detected
 ```
 
+#### Showing additional info on the card
+
+![изображение](https://user-images.githubusercontent.com/71872483/170584118-ef826b60-dce3-42ec-a005-0f467616cd37.png)
```

Comment:
Since I added the other comment on the code. Here are three more:
* There seems to have sneaked in some non-English characters. Possibly from uploading the image?
* Should the image be part of the commit?
* Should the new grouping options be documented? If so, I'd suggest to also going on the caveat that month and year will not be 100% accurate, because they use fixed intervals.